Sources/Private/MFBBindingInfo.h

```diff
@@ -0,0 +1,28 @@
+//
+//  MFBBindingInfo.h
+//  Pods
+//
+//  Created by Nickolay Tarbayev on 28.07.2017.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MFBUnbindable
+- (void)unbind;
+@end
+
+@interface MFBBindingInfo : NSObject
+
++ (instancetype)bindingInfoForObject:(id)obj;
+
+- (void)addBinding:(id<MFBUnbindable>)binding forKey:(NSString *)key group:(NSString *)group;
+- (void)removeBinding:(id<MFBUnbindable>)binding forKey:(NSString *)key group:(NSString *)group;
+
+- (NSArray *)bindingsForKey:(NSString *)key inGroup:(NSString *)group;
```

Comment:
Bindings being added or removed have to be unbindable (`MFBUnbindable`). Bindings returned for key and group pair are just objects without any specific requirement as clients should not unbind them.